fastly-edge-assignments/fastly.toml

```diff
@@ -0,0 +1,17 @@
+# This file describes a Fastly Compute package. To learn more visit:
+# https://www.fastly.com/documentation/reference/compute/fastly-toml
+
+authors = ["engineering@geteppo.com"]
+name = "Eppo Assignments on Fastly Compute@Edge"
+description = "Edge compute service for pre-computed Eppo flag assignments"
+language = "rust"
+manifest_version = 3
+
+[scripts]
+build = "cargo build --bin fastly-edge-assignments --release --target wasm32-wasi --color always"
+
+[local_server]
+  [local_server.kv_stores]
+    [[local_server.kv_stores.edge-assignment-kv-store]]
+      key = "ufc-by-sdk-key-token-hash-V--77TScV5Etm78nIMTSOdiroOh1__NsupwUwsetEVM"
+      file = "../sdk-test-data/ufc/flags-v1.json"
```

Comment:
this instructs the `fastly serve` process to populate the local in-memory KV store with one key, a UFC configuration at a precomputed hash. See the unit test in this repo for the associate SDK key to use.